src/ui/field_formats_mixin.js

```diff
@@ -22,7 +22,7 @@ export function fieldFormatsMixin(kbnServer, server) {
     return new FieldFormatsService(fieldFormatClasses, getConfig);
   });
 
-  server.decorate('server', 'registerFieldFormatClass', (FieldFormat) => {
+  server.decorate('server', 'registerFieldFormat', (FieldFormat) => {
```

Comment:
This does not work. You are saving the function `createFieldFormat` in fieldFormatClasses instead of returned a `FieldFormat` class from the function call.

Reply:
Whoops! Focused my tests on the UI